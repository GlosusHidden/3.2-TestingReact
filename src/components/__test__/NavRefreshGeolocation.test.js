import React from 'react'
import renderer from 'react-test-renderer'

import { Provider } from 'react-redux'

import NavRefreshGeolocation from '../NavRefreshGeolocation'

import {myStore, mockStore} from './test_data/data'

it('NavRefreshGeolocation Component Snapshot', () => {
  const geolocationMock = {
    getCurrentPosition: jest.fn()
  };
  global.navigator.geolocation = geolocationMock;
  const renderedValue = renderer.create(
    <Provider store={mockStore}>
      <NavRefreshGeolocation />
    </Provider>
  ).toJSON();
  expect(renderedValue).toMatchSnapshot();
});
