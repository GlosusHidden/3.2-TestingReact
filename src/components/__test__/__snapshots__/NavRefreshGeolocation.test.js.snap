// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavRefreshGeolocation Component Snapshot 1`] = `
<div
  id="nav_refresh_geolocation"
>
  <div
    className="text"
  >
    Weather here
  </div>
  <button
    className="button"
    id="refresh_geolocation"
    onClick={[Function]}
  >
    Refresh Geolocation
  </button>
</div>
`;
